src/mmw/js/src/modeling/models.js

```diff
@@ -194,9 +194,9 @@ var ScenarioModel = Backbone.Model.extend({
     },
 
     initialize: function() {
-        // TODO: When fetching model or populating from a preloaded source
-        // this attribute should not be initialized as empty.
-        this.set('modifications', new ModificationsCollection());
+        if (this.get('modifications') === null || "[]") {
```

Comment:
Updated so that we can duplicate scenarios and carry along the modifications. However, loading saved modifications is currently not working: #278. Updated so that the code doesn't throw an error.